tools/android/BUILD.tools

```diff
@@ -3,6 +3,9 @@ load("//tools/python:private/version.bzl", "PY_BINARY_VERSION")
 
 package(default_visibility = ["//visibility:public"])
 
+# The toolchain type used to distinguish Android SDK toolchains.
+toolchain_type(name = "sdk_toolchain_type")
```

Comment:
Why not use the regular toolchain_type naming convention? Maybe worth leaving a note explaining so this isn't copied without understanding? 

Is it because this file isn't clearly for just sdk logic so the toolchain type purpose isn't obvious from the target path? That would make sense to me. i guess probably not worth creating an sdk subdir for it?

Reply:
I wanted to clarify that this is specifically for the SDK. I don't think we'll need an NDK toolchain type (that's really just a special case of the C++ toolchain type), but there's also various other things in Android that might be toolchains.

Reply:
Yeah that makes sense to me. I'd say maybe leave a comment that this isn't a conventional toolchain type name (re: https://docs.bazel.build/versions/master/toolchains.html#writing-rules-that-use-toolchains) but up to you

Reply:
I don't think the comment is helpful, there are plenty of other examples to follow.